meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta.inc

```diff
@@ -36,14 +49,22 @@ do_version_check() {
 addtask do_version_check after do_patch before do_install
 
 do_configure() {
-    # We need the sed operation because bitbake misinterprets the JSON closing
-    # brace as end of function, unless we put spaces in front of it.
-    sed -e 's/^ //' <<EOF | cat > ${WORKDIR}/mender-binary-delta.conf
- {
+    echo "{" > ${WORKDIR}/mender-binary-delta.conf
+
+    if [ -n "${MENDER_FLASH_TOOL}" ]; then
```

Comment:
Oh darn! I just realized this is wrong in both branches (master too)!

It needs to check the value, and set either `mender-flash` or `ubiupdatevol`, not just `mender-flash` unconditionally!

Could you fix both?

Reply:
Cherry-picked now :+1: